resource-aggregate/pb/events.proto

```diff
@@ -113,4 +113,20 @@ message ResourceStateSnapshotTaken {
     repeated ResourceDeletePending resource_delete_pendings = 6;
     AuditContext audit_context = 100;
     EventMetadata event_metadata = 101;
+}
+
+message DeviceCloudStatusUpdated {
+    string device_id = 1;
+    OnlineState online = 2;
+    ResourceObservationsState resource_observations = 3;
+    AuditContext audit_context = 100;
+    EventMetadata event_metadata = 101;
+}
+
+message DeviceCloudStatusSnapshotTaken {
+    string device_id = 1;
+    OnlineState online = 2;
+    ResourceObservationsState resource_observations = 3;
```

Comment:
Observation is CoAP specific term. By this name, you describe tech solution hidden behind device/resource shadow. The user is not disabling observations, he doesn't have to know what it is.
`ShadowSync`:`Enabled`/`Disables`